### ISSUE TITLE
Font size on tablet screen

### ISSUE THREAD

**Issue by DarkSide666** [atk4/ui#485]
One of our customers said that his UI interface looks to small on tablet screen.
It's like this.
![screenshot_20180607-125336](https://user-images.githubusercontent.com/1969119/41105262-f6286836-6a75-11e8-89ae-cffb856f964e.jpg)

Solution was like described here: https://css-tricks.com/forums/topic/semantic-ui-font-size/.

Please check this out and maybe we need to add some changes in Semantic-UI stylesheet to make UI more compatible (responsive) with tablet screen.

**Comment by ibelar** [comment]
Viewport need to be setup for mobile device:

Add proper meta tag in your app and it should work fine:

```
$app->html->template->trySetHtml('meta', $app->getTag('meta', ['name' => 'viewport', 'content' => 'width=device-width, initial-scale=1.0']));
````

**Comment by PhilippGrashoff** [comment]
Should be fixed in #608

**Comment by DarkSide666** [comment]
Should be fine now. If not, then we can reopen this issue or create new one.
